### PR TITLE
Bookpage

### PR DESCRIPTION
**Pull request by naveen623**
Book page from Naveen task completed. Ready for Integration

### CODE REVIEW

File: project1/application.py [hunk 1]
```diff
@@ -116,3 +119,26 @@ def userHome(Username):
...
+
+
+
+
+
+
+
+
+@app.route("/bookpage/<isbn>",methods=["GET","POST"])
+def bookpage(isbn):
```
Comment (ashishk144) [inline project1/application.py]: check if the user is logged in or not

File: project1/application.py [hunk 2]
```diff
@@ -116,3 +119,26 @@ def userHome(Username):
...
+	
+	bk = isbn
+
+	book = Books.query.filter_by(isbn=bk).first()
+	
+	if request.method == "POST":
```
Comment (ashishk144) [inline project1/application.py]: why is a POST request required for bookpage?

File: project1/templates/book.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+
+<html class="no-js">
+
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```
Comment (ashishk144) [inline project1/templates/book.html]: Use a generic template and extend this file to it.

**Review by ashishk144 — COMMENTED** [review]
Work on the feedback provided

File: project1/application.py (hunk 1 above)
Comment (naveen623) [reply project1/application.py]: Added functionality

File: project1/templates/book.html (hunk 3 above)
Comment (naveen623) [reply project1/templates/book.html]: Layout added and extended

File: project1/application.py (hunk 2 above)
Comment (naveen623) [reply project1/application.py]: It is required to bring the review details to the book page.

**Comment by naveen623** [comment]
All the chages are made accordingly.

**Pull request closed without merging**